index.html

```diff
@@ -25,70 +25,310 @@
+                          <a href="https://github.com/Haywayaheadshot" target="_blank" rel="noopener noreferrer"><i class="fa fa-github"></i></a>
+                      </dt>
+                      <dd></dd>
+                      <dt class="list-item">
+                          <a href="https://angel.co/u/abubakar-ummar" target="_blank" rel="noopener noreferrer"><i class="fa fa-angellist"></i></a>
+                      </dt>
+                      <dd></dd>
+                      
+                  </dl>
+                  
+                  </section>
+            </main>
+      </div>
+      <div class="portfolio-container page-one">
+        <header class="pageone-header">
+          <img src="images/SnapshootPortfolio.svg" alt="A snapshoot Portfolio Header From Figma">
+        </header>
+        <section class="pageone-body">
+            <h1><strong>Tonic</strong></h1>
+            <section>
+              <h4 class="subhead">CANOPY</h4>
+              <img src="images/Counter.svg" alt="counter shape"><h6 class="subhead">Front End Dev</h6>
+              <img src="images/Counter.svg" alt="counter shape"><h6 class="subhead">2022</h6>
+            </section>
+            <section class="pageone-text">
+              <div>daily selection of privately personalized reads; no accounts or sign-ups required.</div>
+            </section>
+            <section class="buttons-section">
+              <ul >
+                <li class="js-css-html-button-ul">
+                  <button type="button" class="html-css-js-button">html</button>
+                </li>
+                <li class="js-css-html-button-ul">
+                  <button type="button" class="html-css-js-button">css</button>
+                </li>
+                <li class="js-css-html-button-ul">
+                  <button type="button" class="html-css-js-button">javascript</button>
+                </li>
+              </ul>
+            </section>
+            <section>
+              <button type="button" class="footer-button">See Project</button>
+            </section>
+        </section>
+      </div>
+       <div class="portfolio-container page-two">
+        <header class="pageone-header">
+          <img src="images/SnapshootPortfoliopage2.svg" alt="A snapshoot Portfolio Header From Figma">
+        </header>
+        <section class="pageone-body">
+          
+            <h2 class="page-two-text"><strong>Multi-Post</strong></h2>
+            <h2 class="page-two-text"><strong>Stories</strong></h2>
+          
+            <section>
+              <h4 class="subhead">CANOPY</h4>
+              <img src="images/Counter.svg" alt="counter shape"><h6 class="subhead">Front End Dev</h6>
+              <img src="images/Counter.svg" alt="counter shape"><h6 class="subhead">2022</h6>
+            </section>
+            <section class="pageone-text">
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </section>
```

Comment:
- [ ] Kindly wrap this information part of the project into the `p` tag. 😉 Remember to change this on other cards as well.